### PR TITLE
fix(INJI-285): Added the flip camera buton in the scan screen

### PR DESCRIPTION
**Pull request by Anil-kumar-Majji**
[**Inji-285**](https://mosip.atlassian.net/browse/INJI-285) -->  **The flip camera button is missing on the scan screen**

### CODE REVIEW

File: components/ui/themes/PurpleTheme.ts [hunk 1]
```diff
@@ -965,6 +965,7 @@ export const PurpleTheme = {
   ProfileIcon: require('../../../purpleAssets/profile_icon.png'),
   MosipSplashLogo: require('../../../assets/icon.png'),
   MosipLogo: require('../../../assets/mosip-logo.png'),
+  CameraFlipIcon: require('../../../assets/camera-flip-icon.png'),
```
Comment (vharsh) [inline components/ui/themes/PurpleTheme.ts:968]: Should this require a different coloured icon considering the background colour differences in the UI?
IMO, this should not increase app-bundle size as only one png will be used in one theme.

cc @swatigoel

**Review by vharsh — COMMENTED** [review]
minor suggestions

File: components/ui/themes/PurpleTheme.ts (hunk 1 above)
Comment (Anil-kumar-Majji) [reply components/ui/themes/PurpleTheme.ts:968]: For now, they provided an orange-white [wire-frame](https://xd.adobe.com/view/21e1c3f7-f487-4f20-a28f-d292c1adba70-a/screen/5e4c79f6-dcd7-4900-aea2-670e637b4995/). Just to not keep this part with empty UI I used the orange one. If once I get the purple wire frame along with the access to downloading png assets, I will replace this.

**Review by vharsh — APPROVED** [review]
lgtm

**Pull request merged**